src/Cameras/babylon.freeCamera.ts

```diff
@@ -95,23 +95,11 @@
 
         // Controls
         public attachControl(element: HTMLElement, noPreventDefault?: boolean): void {
-            if (this._attachedElement) {
-                return;
-            }
-            this._noPreventDefault = noPreventDefault;
-            this._attachedElement = element;
-            noPreventDefault = Camera.ForceAttachControlToAlwaysPreventDefault ? false : noPreventDefault;
-
             this.inputs.attachElement(element, noPreventDefault);

```

Comment:
This should be this.inputs.attachElement(element, Camera.ForceAttachControlToAlwaysPreventDefault ? false : noPreventDefault)
I'll fix it for my next commit